pkg/metallb/bgppeer.go

```diff
@@ -261,6 +261,32 @@ func (builder *BGPPeerBuilder) Update(force bool) (*BGPPeerBuilder, error) {
 	return builder, err
 }
 
+// WithDynamicASN defines the remoteAS as either internal (iBGP) or external (eBGP).
```

Comment:
```suggestion
// WithDynamicASN defines the DynamicASN as either internal (iBGP) or external (eBGP).
```

Reply:
Maybe add also inforamtion that it's mutually exclusive with ASN

Reply:
Made the change and added info.